src/services/binance/service.ts

```diff
@@ -1,18 +1,27 @@
-import { WS } from '@thorchain/asgardex-binance'
-import { map } from 'rxjs/operators'
+import * as RD from '@devexperts/remote-data-ts'
+import { WS, Client } from '@thorchain/asgardex-binance'
+import { right, left } from 'fp-ts/lib/Either'
+import * as E from 'fp-ts/lib/Either'
+import * as FP from 'fp-ts/lib/function'
+import { none, some } from 'fp-ts/lib/Option'
+import * as O from 'fp-ts/lib/Option'
+import * as Rx from 'rxjs'
+import { Observable, Observer } from 'rxjs'
+import { map, mergeMap, tap, filter, catchError, retry, shareReplay, concatMap } from 'rxjs/operators'
 import { webSocket } from 'rxjs/webSocket'
 
 import { envOrDefault } from '../../helpers/envHelper'
+import { observableState, triggerStream } from '../../helpers/stateHelper'
+import { KeystoreState } from '../wallet/types'
+import { getPhrase } from '../wallet/util'
+import { BalancesRD, BinanceClientStateForViews, BinanceClientState } from './types'
 
 const BINANCE_TESTNET_WS_URI = envOrDefault(
   process.env.REACT_APP_BINANCE_TESTNET_WS_URI,
   'wss://testnet-dex.binance.org/api/ws'
 )
 
-export const BINANCE_MAINET_WS_URI = envOrDefault(
-  process.env.REACT_APP_BINANCE_MAINNET_WS_URI,
-  'wss://dex.binance.org/api/ws'
-)
+const BINANCE_MAINET_WS_URI = envOrDefault(process.env.REACT_APP_BINANCE_MAINNET_WS_URI, 'wss://dex.binance.org/api/ws')
```

Comment:
Would it be worth storing these defaults in one place?

Reply:
Good catch: In this case we just need it in `services/binance/service.ts`